dcos/dcos-ui
# DCOS-41059: Deleting groups in Services has a misleading message

Added "or groups" to make delete message more accurate.

Closes https://jira.mesosphere.com/browse/DCOS-41059

## Testing
1. Go to services tab
2. Create a group
3. Create a group inside that group
4. Try to delete the first group
5. See the message that appears.

## Trade-offs
None.

## Dependencies
None.

## Screenshots

### Before
![screenshot from 2018-12-07 12-21-14](https://user-images.githubusercontent.com/40791275/49642042-9d4d9080-fa1a-11e8-86a4-c1bd43844599.png)

### After
![screenshot from 2018-12-07 11-45-21](https://user-images.githubusercontent.com/40791275/49642013-8eff7480-fa1a-11e8-9509-04fa405774c8.png)
